### ISSUE TITLE
[BUG]: Title is null when sidecar text tracks loaded

### ISSUE THREAD

**Issue by mkbhdana** [TheWidlarzGroup/react-native-video#3663]
### Version

v6 (Beta)

### What platforms are you having the problem on?

Android

### Architecture

Old architecture

### What happened?

A bug happened! 
using external subtitles sets title value to null even though title string is provided

### Reproduction

repository link

### Reproduction

Step to reproduce this bug are: 

add external subtitle as per docs and console log textTrack  title will be null

![image](https://github.com/react-native-video/react-native-video/assets/85346255/9972b4aa-ad97-4b29-ad77-936d85b31d15)

![image](https://github.com/react-native-video/react-native-video/assets/85346255/28a99598-e810-46a1-b07f-7edc84f1873b)

**Comment by mkbhdana** [comment]
@KrzysztofMoch please help in this case as i am not able to set subtitle as i rely on title type. Index throws error thats why can't switch to that and also i recieve same mutiple language

**Comment by KrzysztofMoch** [comment]
@freeboub I was able to fix this by changing one line in `ReactExoplayerView.java` in `exoplayerTrackToGenericTrack` function
```java
private Track exoplayerTrackToGenericTrack(Format format, int trackIndex, TrackSelection selection, TrackGroup group) {
        Track track = new Track();
        track.setIndex(trackIndex);
        if (format.sampleMimeType != null) track.setMimeType(format.sampleMimeType);
        if (format.language != null) track.setLanguage(format.language);
        // Fixed - we are accessing label as we set title to label
        if (format.label != null) track.setTitle(format.label);
        // Old - we are accessing id and it is null
        // if (format.id != null) track.setTitle(format.id);
        track.setSelected(isTrackSelected(selection, group, 0));
        return track;
}
```

But I have question we should move setter to set tittle to `id` or getter to get title from `label` ?

```java
private MediaSource buildTextSource(String title, Uri uri, String mimeType, String language) {
        MediaItem.SubtitleConfiguration subtitleConfiguration = new MediaItem.SubtitleConfiguration.Builder(uri)
                .setMimeType(mimeType)
                .setLanguage(language)
                .setSelectionFlags(C.SELECTION_FLAG_DEFAULT)
                .setRoleFlags(C.ROLE_FLAG_SUBTITLE)
                // we are setting title to "Label" not "Id" 
                .setLabel(title)
                .build();
        return new SingleSampleMediaSource.Factory(mediaDataSourceFactory)
                .createMediaSource(subtitleConfiguration, C.TIME_UNSET);
    }
```

**Comment by KrzysztofMoch** [comment]
Fix will be available in `beat.9`

**Comment by KrzysztofMoch** [comment]
@mkbhdana we released `6.0.0-rc.0` that include fix for this
